src/test/java/se/sundsvall/invoicesender/integration/db/entity/BatchEntityTests.java

```diff
@@ -56,6 +59,7 @@ void testBuilderPattern() {
 		Assertions.assertThat(batchEntity.getIgnoredItems()).isEqualTo(123L);
```

Comment:
Replace with just `.assertThat(...)`, since you already have the static import for it? The same goes for all `Assertions.assertThat(...)`.